libcaja-private/caja-file-operations.c

```diff
@@ -5966,10 +5962,14 @@ caja_file_operations_copy_move (const GList *item_uris,
 					       done_callback, done_callback_data);
 	}
 
-    	g_list_free_full (locations, g_object_unref);
+    g_list_free_full (locations, g_object_unref);
```

Comment:
This line has not been modified. Could you leave it as it was before?

Reply:
That padding is gonna haunt me. :( Caja is overdue for some serious linting. Rolled back, as it has nothing to do with this PR.